spec/factories/approvals.rb

```diff
@@ -1,10 +1,10 @@
 FactoryGirl.define do
```

Comment:
Good question! At present _nothing_ uses the new `Step` factory, everything's still using `Approval`. I'm fine with that for the moment, since you can't just create a `Step` without specifying what kind of step it is. In the future we should probably rearrange things so that you always create a `Step` with an associated trait, but we don't have to do that right now.